src/components/learning-units-section/complete-learning-unit-toogle/CompleteLearningUnitToggleAuth.js

```diff
@@ -0,0 +1,41 @@
+import useGet from '@hooks/useGet';
+import { endpoints } from '@utils/endpoints';
+import { Skeleton } from 'primereact/skeleton';
+import { CompleteLearningUnitToggleButton } from './CompleteLearningUnitToggleButton';
+
+export const CompleteLearningUnitToggleAuth = ({ unit, showSuccess }) => {
+  const completedLearningUnitEndpoint = endpoints(
+    'isLearningUnitCompleted',
+    unit.id
+  );
+  const {
+    data: isCompleted,
+    isLoading,
+    isError,
+    mutate,
+  } = useGet(completedLearningUnitEndpoint);
+  const changeHandler = (clicked) => {
+    const requestOptions = {
+      method: clicked.value ? 'POST' : 'DELETE',
+      headers: { 'Content-Type': 'application/json' },
+    };
+    fetch(completedLearningUnitEndpoint, requestOptions).then((response) => {
```

Comment:
please don't use `.then()`. 

Use `async` and `await` 👍 (o cambiarlo si fue de alguien otro)

Reply:
Done!